app/controllers/application_create.py

```diff
@@ -1,115 +1,144 @@
-    # else:
-    #     flash("Your application was not submitted, for an error occured in the process.")
-    #     return redirect(url_for("/application/create/<gid>"))
-    
+            if 'cv' in request.files:
+                # retrieve the cv uploaded in the application form
+                cv             = request.files['cv']
+                
+                # store the file extension 
+                cv_ext         = (str(cv.filename.split(".").pop())).replace(" ","")
+                
+                # rename the file uploaded to a specific format
+                cv_filename    = "cv_{}".format(data['firstName']+ '_'+ data['lastName'])
+                
+                # store the absolute path where the file will be stored on the server
+                cv_upload_path = getAbsolutePath(cfg['paths']['files'],cv_filename,True)
+                
+                # save the file on the server
+                cv.save(cv_upload_path)
+                
+                # save the file in the database in the Files table
+                application_obj.insert_attachment_file("cv", fid,cv_filename,cv_upload_path,cv_ext)
         
+        except Exception as e:
+            print (e)
         
+            flash("An error occured while saving cv file!")
+        
+            return render_template('views/application_create.html',gid=gid, gallery = gallery, cfg=cfg)
+        
+        try:    
+        
+            if 'statement' in request.files:   
+                # retrieve the personal statement uploaded in the application form 
+                statement             = request.files['statement']
+                
+                # store the file extension 
+                statement_ext         = (str(statement.filename.split(".").pop())).replace(" ","")
+                
+                # rename the file uploaded to a specific format
+                statement_filename    = "personal_statement_{}".format(data['firstName']+ '_'+ data['lastName'])
+                
+                # store the absolute path where the file will be stored on the server
+                statement_upload_path = getAbsolutePath(cfg['paths']['files'],statement_filename,True)
+                
+                # save the file on the server
+                statement.save(statement_upload_path)
+                
+                # save the file in the database in the Files table
+                application_obj.insert_attachment_file("statement", fid,statement_filename,statement_upload_path,statement_ext)
+        
+        except Exception as e:
+            print (e)
+        
+            flash("An error occured while saving the personal statement file!")
+        
+            return render_template('views/application_create.html',gid=gid, gallery = gallery, cfg=cfg)
+        print("Wait!")   
+        try:
+            number = 1
+            images = request.files['fileDropZone']
+            print("Is it it here?")
```

Comment:
I know you are still working though this, but just a reminder to please remove print statments when the code is finished